concepts/list-methods/about.md

```diff
@@ -1,12 +1,12 @@
 # About
 
-In Python, a [`list`][list std type] is a mutable collection of items in _sequence_. Like most collections (see the built-ins tuple, dict and set), lists can hold reference to any (or multiple) data type(s) - including other lists.
+A [`list`][list] is a mutable collection of items in _sequence_. Like most collections (_see the built-ins [`tuple`][tuple], [`dict`][dict] and [`set`][set]_), lists can hold reference to any (or multiple) data type(s) - including other lists.  They can be copied in whole or in part via [slice notation][slice notation]. Like any [sequence][sequence type], elements within `lists` are referenced by `0-based index` number.
```

Comment:
```suggestion
A [`list`][list] is a mutable collection of items in _sequence_. Like most collections (_see the built-ins [`tuple`][tuple], [`dict`][dict] and [`set`][set]_), lists can hold references to any (or multiple) data type(s) - including other lists.  They can be copied in whole or in part via [slice notation][slice notation]. Like any [sequence][sequence type], elements within `lists` are referenced by `0-based index` number.
```

Reply:
Does it make sense to call out when lists are used vs tuples? Or that lists are usually of a uniform type?

Reply:
🤔   I did have an earlier iteration of this that used the terms heterogeneous and homogeneous for data...but worried that would lead to confusion, since homogeneity of _data type_ is not a requirement of either container, even though _use_ for `lists` tends to be homogeneous.  Any thoughts or suggestions on how you'd do that?  Here is what I did for `tuples`:  https://github.com/exercism/python/blob/main/concepts/tuples/about.md#tuples-as-related-information

Reply:
> By convention, lists are used for collections of items when the items are all similar and there can be any number of those items - like scores or items in a shopping basket. In contract, tuples are used when a piece of data has multiple pieces, like an `(x, y)` or a polar coordinate - `(radius, degree)`.

Reply:
I like that.  Second question:  do we want that in this `about.md` (which is for list-methods) -- or should we be putting it in the `about.md` for lists?

Reply:
`lists` seems to be the basics without too many methods and `list-methods` seems like the extended method list. As such, I think `list-methods` makes more sense.